metrics/src/metrics.rs

```diff
@@ -337,7 +337,6 @@ impl MetricsAgent {
     }
 
     pub fn submit(&self, mut point: influxdb::Point, level: log::Level) {
-        point.add_tag("host_id", influxdb::Value::String(HOST_ID.to_string()));
```

Comment:
This tag was erroneously added as part of https://github.com/solana-labs/solana/pull/5291